### PR TITLE
SASS-7859 - API for SBA

### PR DESCRIPTION
**Pull request by willjshark**

### CODE REVIEW

**Review by modivilioglu — APPROVED** [review]
LGTM

**Comment by arunkumar461** [comment]
Journey name should be `SBA`

image

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

File: conf/app.routes [hunk 1]
```diff
@@ -36,4 +36,6 @@ POST         /property/:taxYear/:businessId/expenses/:nino/:incomeSourceId/answe
 
 PUT         /property/:taxYear/:businessId/expenses/:nino/:incomeSourceId/answers/:submissionId                       controllers.JourneyAnswersController.updateExpenses(taxYear: TaxYear, businessId: BusinessId, nino: Nino, incomeSourceId: IncomeSourceId, submissionId: SubmissionId)
 
-PUT         /property/:taxYear/:businessId/income/:nino/:incomeSourceId/esba/answers                      controllers.JourneyAnswersController.updateEsba(taxYear: TaxYear, businessId: BusinessId, nino: Nino, incomeSourceId: IncomeSourceId)
\ No newline at end of file
+PUT         /property/:taxYear/:businessId/income/:nino/:incomeSourceId/esba/answers                      controllers.JourneyAnswersController.updateEsba(taxYear: TaxYear, businessId: BusinessId, nino: Nino, incomeSourceId: IncomeSourceId)
+
+PUT         /property/:taxYear/:businessId/income/:nino/:incomeSourceId/sba/answers                      controllers.JourneyAnswersController.updateSba(taxYear: TaxYear, businessId: BusinessId, nino: Nino, incomeSourceId: IncomeSourceId)
```
[the remark this answers is not in the document]
Comment (willjshark) [reply conf/app.routes]: Fixed

File: app/controllers/RequestHandler.scala [hunk 2]
```diff
@@ -36,9 +36,10 @@ trait RequestHandler {
                                       taxYear: TaxYear,
                                       businessId: BusinessId,
                                       nino: Nino,
+                                      journeyName: JourneyName,
```
Comment (modivilioglu) [inline app/controllers/RequestHandler.scala:39]: Nice

**Review by modivilioglu — APPROVED** [review]
LGTM

**Pull request merged**